Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/Match.java

```diff
@@ -0,0 +1,27 @@
+package org.cqframework.cql.cql2elm.model;
+
+public enum Match {
+    CASE, CASE_IGNORED, SOUNDS_LIKE, NONE;
+
+    public static Match checkMatch(String val, String checkVal) {
```

Comment:
The name of this function, to me at least, implies that it will throw an error if there is no match. Suggest naming this something that more clearly indicates this is getting the match type. Say `getMatchType()`?